sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/SearchIndexerClientBuilder.java

```diff
@@ -36,40 +36,126 @@
 
 /**
  * This class provides a fluent builder API to help aid the configuration and instantiation of {@link
- * SearchIndexerClient SearchIndexerClients} and {@link SearchIndexerAsyncClient SearchIndexerAsyncClients}. Call {@link
- * #buildClient() buildClient} and {@link #buildAsyncClient() buildAsyncClient} respectively to construct an instance of
- * the desired client.
+ * SearchIndexerClient SearchIndexerClients} and {@link SearchIndexerAsyncClient SearchIndexerAsyncClients}.
+ *
+ * <h2>
+ *     Overview
+ * </h2>
+ *
+ * <p>
+ *     This client allows you to create instances of {@link SearchIndexerClient} and {@link SearchIndexerAsyncClient} to
+ *     utilize synchronous and asynchronous APIs respectively to interact with Azure AI Search.
+ * </p>
+ *
+ * <h2>
+ *     Getting Started
+ * </h2>
+ *
+ * <h3>
+ *     Authentication
+ * </h3>
+ *
+ * <p>
+ *     Azure AI Search supports <a href="https://learn.microsoft.com/en-us/azure/search/search-security-rbac?tabs=config-svc-portal%2Croles-portal%2Ctest-portal%2Ccustom-role-portal%2Cdisable-keys-portal">
+ *         Microsoft Entra ID (role-based) authentication </a> and <a href="https://learn.microsoft.com/en-us/azure/search/search-security-api-keys?tabs=portal-use%2Cportal-find%2Cportal-query">API keys</a> for authentication.
```

Comment:
Overall need to remove all `en-us` URL path segments to allow the browser to infer the language locale of the user following the link

Reply:
I tried to filter them out of the links but I missed a few.

Reply:
Thank you, I'll get the rest of them.